sopel/cli/utils.py

```diff
@@ -39,7 +39,7 @@ def _colored(text, color, reset=True):
 
 
 def green(text, reset=True):
-    """Add ANSI escape sequences to make the text green in term
+    """Add ANSI escape sequences to make the text green in term.
```

Comment:
I'm going to add a batch of suggestions all related to this so you can apply (or ignore) them all easily.

Seems like something I should have flagged when these went in in the first place, but oh well.

```suggestion
    """Add ANSI escape sequences to make the text green in terminal.
```